projects/m3/001-total-the-values/js/index.js

```diff
@@ -0,0 +1,19 @@
+// const prompt = require("prompt-sync")({ sigint: true}) ;
+
+function sumAskedNumbers(){
+    const askedNumber = prompt('Enter a number');
+
+    if(askedNumber === ''){
+        return '0.0';
+    }
+    
+    return(parseFloat(askedNumber) + parseFloat(sumAskedNumbers()));

```

Comment:
Couldn't it be better to convert immediately the entered number by the user and return a float or int instead of working with strings?

Reply:
Fixed it.